### DISCUSSION TITLE
Show that the app is doing somthing.

### DISCUSSION THREAD

**Discussion by KingOpB** [brentvollebregt/auto-py-to-exe#369] [Ideas]
Kindly may I suggest that a loading bar or circle, be added so that way it shows that the application is still doing something rather than it looks like it has frozen.

**Comment by brentvollebregt** [comment]
Is this for during packaging or some other time? The output in the UI shows progress and until it shows signs of being complete (says it's done or an error has occurred), packaging is still running.
